docs/docusaurus.config.js

```diff
@@ -82,6 +82,14 @@ const config = {
       '@docusaurus/plugin-client-redirects',
       {
         redirects: [
+          {
+            to: '/',
```

Comment:
Looks like you're missing something in this redirect.

Reply:
This will go to `/` here
https://github.com/langflow-ai/langflow/pull/4547/files#diff-80f4dc5df930fce44ea562dc54bac63be657f662c422fde095eb9f2ddac2a0cdR4